src/libs/modulegroups.c

```diff
@@ -1523,7 +1523,6 @@ void init_presets(dt_lib_module_t *self)
 
   SMG(C_("modulegroup", "base"), "basic");
   AM("basecurve");
-  AM("clipping");
```

Comment:
I would have put "ashift" here (and remove it from it's actual group) as it now handle rotation which belong to the basic group imho (same for other presets)